Update dec. article 5-Update-the-variant-axis-of-your-product-models.md (#878)
akeneo/pim-helpcenter: content/updates/2020-12/5-Update-the-variant-axis-of-your-product-models.md

- EE
---

> # Update the variant axis value of your products

Reviewer: ```suggestion
# Update the axis value of your variant products
```

::: meta-data type="Improvement" features="Governance" available="late December"

> Sometimes, when new variant products are created, their variant axis can be set to a value that is not precisely the one you wanted. It is now possible to change this value afterward, making your product modeling even more flexible. Simply edit the axis value in your product variant form. And you already know this: under-control product information governance is the key to unlock powerful product experiences. :wink:

Reviewer: ```suggestion
Sometimes, when new variant products are created, their variant axis can be set to a value that is not precisely the one you wanted. It is now possible to change this value afterward, making your product modeling even more flexible. Simply edit the axis value in your variant product form. And you already know this: under-control product information governance is the key to unlock powerful product experiences. :wink:
```

Author: ```suggestion
Because your catalog evolves, and variant product axis may change over time. It is now possible to change this value afterwards, making your product modeling even more flexible. Simply edit the axis value in your variant product form. And you already know this: under-control product information governance is the key to unlock powerful product experiences. :wink:
```
